xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2239,7 +2239,7 @@ admin.colortheme.wikiSetting=The color theme configured at the wiki level is {0}
 admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
-admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to use the Extension Manager to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
+admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more. <a href="{0}">Click here to find the information how to install it.</a>.
```

Comment:
I would put the link on "install a flavor". In general I try to never have some text like "click here" when writing HTML. It's a bit unwieldy and doesn't flow too well. WDYT?